index.js

```diff
@@ -1,3 +1,11 @@
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+/* Add Book */
 const addBook = () => {
   let books;
   const bookTitle = document.querySelector('#title').value;
```

Comment:
- [ ] According to this requirement 👇🏿:

  > - Create class methods to add and remove books.

  The methods for adding and removing books should be class methods. So, you ought to have refactored them into the class. Please ensure the `addBook` method is a class method.